doc/flatpak-metadata.xml

```diff
@@ -253,6 +268,54 @@
                                 Available since 0.3.
                             </para></listitem></varlistentry>
 
+                            <varlistentry><term><option>host-os</option></term>
+                            <listitem><para>
+                                The host operating system's libraries,
+                                executables and static data from
+                                <filename>/usr</filename>
+                                and the related directories
+                                <filename>/bin</filename>,
+                                <filename>/lib</filename>,
+                                <filename>/lib32</filename>,
+                                <filename>/lib64</filename>,
+                                <filename>/sbin</filename>.
+                                Additionally, this keyword provides access
+                                to a subset of <filename>/etc</filename> that
+                                is associated with packaged libraries and
+                                executables, even if the
```

Comment:
I've tried to phrase this in a way that would let us expose more of `/etc` as part of `host-os` in future if we want to, without it being considered to be a bug.